crates/router/src/connector/tsys.rs

```diff
@@ -225,9 +194,25 @@ impl ConnectorIntegration<api::PSync, types::PaymentsSyncData, types::PaymentsRe
     fn get_url(
         &self,
         _req: &types::PaymentsSyncRouterData,
-        _connectors: &settings::Connectors,
+        connectors: &settings::Connectors,
     ) -> CustomResult<String, errors::ConnectorError> {
-        Err(errors::ConnectorError::NotImplemented("get_url method".to_string()).into())
+        Ok(format!(
+            "{}servlets/Transnox_API_server",
```

Comment:
Could the difference in case here and on line 117 (and elsewhere) cause any problems?

https://github.com/juspay/hyperswitch/blob/b677da777dbf2c13ccbea2327190c3c92c631f32/crates/router/src/connector/tsys.rs#L117

Reply:
@SanchithHegde I asked him to take it in separate PR.